proto/primitives/astria/primitive/v1/types.proto

```diff
@@ -55,3 +55,10 @@ message Address {
   reserved 1;
   reserved "inner";
 }
+
+// A `TransactionId` is a unique identifier for a transaction.
+// It contains the hash of the transaction, to be included in
+// rollup deposit events for source tracking.
+message TransactionId {
+  string hash = 1;
```

Comment:
Why string over `bytes`? So for identifiers (like rollup ID) that are `bytes` (and hence use base64 encoding).

Reply:
Discussed offline: in cosmos land hex encoded identifiers are standard for identifying transactions. Furthermore, base64 poses issues in URLs. So using `string` here is ultimately a nod toward end users.